src/userscores/qml/MuseScore/UserScores/internal/TimeSignatureSettings.qml

```diff
@@ -38,8 +38,8 @@ FlatButton {
     TimeSignatureView {
         id: timeSignatureView
 
-        anchors.horizontalCenter: root.horizontalCenter
-        anchors.verticalCenter: root.verticalCenter
+        anchors.horizontalCenter: parent.horizontalCenter
```

Comment:
makes sense to use anchors.centeredIn: parent

Reply:
Maybe, I just fixed, what I broken